content/cloud-docs/api-docs/providers.mdx

```diff
@@ -336,6 +726,559 @@ curl \
+    "attributes": {
+      "name": "aws",
+      "namespace": "hashicorp",
+      "version-statuses": [
+        {
+          "version": "3.1.1"
+        }
+      ],
+      "created-at": "2022-02-11T19:16:59.533Z",
+      "updated-at": "2022-02-11T19:16:59.533Z",
+      "registry-name": "private",
+      "permissions": {
+        "can-delete": true
+      }
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organizations"
+        }
+      },
+      "versions": {
+        "data": [
+          {
+            "id": "provver-y5KZUsSBRLV9zCtL",
+            "type": "registry-provider-versions"
+          }
+        ],
+        "links": {
+          "related": "/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws"
+        }
+      }
+    },
+    "links": {
+      "self": "/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws"
+    }
+  }
+}
+```
+
+### Sample Response (public provider)
+
+```json
+{
+  "data": {
+    "id": "prov-fZn7uHu99ZCpAKZJ",
+    "type": "registry-providers",
+    "attributes": {
+      "name": "aws",
+      "namespace": "hashicorp",
+      "registry-name": "public",
+      "created-at": "2020-07-09T19:36:56.288Z",
+      "updated-at": "2020-07-09T20:16:20.538Z",
+      "permissions": {
+        "can-delete": true
+      }
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "my-organization",
+          "type": "organizations"
+        }
+      }
+    },
+    "links": {
+      "self": "/api/v2/organizations/my-organization/registry-providers/public/hashicorp/aws"
+    }
+  }
+}
+```
+
+## Delete a Provider
+
+`DELETE /organizations/:organization_name/registry-providers/:registry_name/:namespace/:name`
+
+### Parameters
+
+| Parameter            | Description                                                                                                                                                                                                  |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `:organization_name` | The name of the organization to delete a provider from. The organization must already exist, and the token authenticating the API request must belong to the "owners" team or a member of the "owners" team. |
+| `:namespace`         | The namespace of the provider that will be deleted.                                                                                                                                                          |
+| `:name`              | The name of the provider that will be deleted.                                                                                                                                                               |
```

Comment:
Unsure if it's worth mentioning if the delete will delete the provider and all of it's associated versions and platform binaries 🤷‍♀️